sdk/nodejs/provider/experimental/analyzer.ts

```diff
@@ -435,3 +447,9 @@ function symbolTableToSymbols(table: typescript.SymbolTable): typescript.Symbol[
     });
     return symbols;
 }
+
+function debug(label: string, msg: any) {
```

Comment:
Can be removed?

Reply:
Whoops, yeah this should not be here :facepalm: